crates/matrix-sdk/src/encryption/recovery/mod.rs

```diff
@@ -629,7 +629,7 @@ impl Recovery {
         }
     }
 
-    #[instrument]
+    #[instrument(skip(self, response))]
```

Comment:
```suggestion
    #[instrument(skip_all)]
```

https://docs.rs/tracing/latest/tracing/attr.instrument.html#skipping-fields